pages/workspace/archive-and-restore.mdx

```diff
@@ -30,4 +30,4 @@ When an item is archived it is moved from your tracking plan to an archive store
   />
 </center>
 
-Find the archive activity in <a href="https://www.avo.app/schemas/default">the Activity Log of your workspace</a>, click the `...` menu on the item and select "Restore".
+Find the archive activity in <a href="https://www.avo.app/schemas/default">the Activity Log of your workspace</a>, click the `...` menu on the item and select "Restore". Make sure you are on the branch the item was archived, both the workspace itself and the activity log.
```

Comment:
> Should we add a disclaimer about the branch needing to be open?
Yes!

> Do you think it would be more likely to be successful if we skip the activity log mention?
Yeah, that is probably a good call

Reply:
Do you think this is clearer?